lib/dai-plugin-mcd/src/ManagedCdp.js

```diff
@@ -22,7 +21,7 @@ export default class ManagedCdp {
       .get(ServiceRoles.CDP_TYPE)
       .getCdpType(this.currency);
     const { rate } = await cdpType.ilkInfo();
-    const art = Maker.DAI.wei((await this._urnInfo()).art);
```

Comment:
I think art * Ilk.rate is the actual debt value, as @jparklev had mentioned before?  I just realized I also need to update the way `getTotalDebt` is calculated in CdpType to multiply Art by Ilk.rate...

Reply:
I multiply by rate below

Reply:
oops - guess I was projecting haha